tests/rest/admin/test_user.py

```diff
@@ -2080,6 +2239,36 @@ def _check_fields(self, content):
             self.assertIn("quarantined_by", m)
             self.assertIn("safe_from_quarantine", m)
 
+    def _order_test(
+        self,
+        order_type: Optional[str],
+        expected_media_list: List[str],
+        dir: Optional[str] = None,
```

Comment:
Can you re-order these input parameters as `order_type`, `dir`, then `expected_media_list` since the first two get used together.

It also probably makes more sense to rename `order_type` to `order_by` to match the server parameter.

Reply:
Do you have a hint for me? Now I get this error in CI after change the positions (re-order)):
`tests/rest/admin/test_user.py:2243:8: E999 SyntaxError: non-default argument follows default argument`

Reply:
Ahhhh, OK you can't reorder them this way due to it having a default of `None`, maybe doing `expected_media_list`, then `order_by`, then `dir` would make sense. 👍  This way `dir` can still default to `None`.